components/accordion/stories/accordion.stories.js

```diff
@@ -180,33 +112,109 @@ WithForcedColors.parameters = {
 };
 
 // ********* DOCS ONLY ********* //
+
+/**
+ * Accordion items have a default width for each size, but a custom width can also be set to any
+ * width that meets or exceeds the minimum width.
+ */
+export const CustomWidth = AccordionGroup.bind({});
+CustomWidth.tags = ["!dev"];
+CustomWidth.storyName = "Custom width";
+CustomWidth.args = {
+	items: accordionContent,
+	customStyles: {
+		"--mod-accordion-item-width": "500px",
+	},
+};
+CustomWidth.parameters = {
+	chromatic: { disableSnapshot: false },
+};
+
+/**
+ * The compact density has less vertical spacing between rows.
+ */
 export const Compact = Template.bind({});
 Compact.tags = ["!dev"];
 Compact.args = {
-	items: content,
+	items: accordionContent,
 	density: "compact",
 };
 Compact.parameters = {
 	chromatic: { disableSnapshot: true },
 };
-Compact.storyName = "Density - Compact";
+Compact.storyName = "Density: Compact";
```

Comment:
If we prefer sentence-case, would this be "Density: compact" even with the colon? Does the "sentence-case" rule  change in this case? I don't actually know hahah 🤷‍♀️

I don't think it's technically blocking anyways, just something I noticed. If you decided to change this one, we should do the same for the spacious story name.